logexp/README.md

```diff
@@ -0,0 +1,40 @@
+# LogExp
```

Comment:
We should consider renaming it. "LogExp" for me is associating with "logarithm" and "exponential" (which has nothing to do with it).
Maybe just "LogExporter" ?